### ISSUE TITLE
Couldn't build with MySQL 5.7.12

### ISSUE THREAD

**Issue by cosmo0920** [mroonga/homebrew-mroonga#6]
```log
make[2]: Nothing to be done for `all'.
/bin/sh ./libtool  --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I.  -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/sql -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/extra/rapidjson/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/regex -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/libbinlogevents/export -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/libbinlogevents/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12 -I/usr/local/Cellar/mysql/5.7.12/include/mysql -I/usr/local/Cellar/groonga/6.0.1/include/groonga -I./lib   -I/usr/local/Cellar/mysql/5.7.12/include/mysql -fno-omit-frame-pointer -DMYSQL_DYNAMIC_PLUGIN -g -O2 -Wall -Wextra -Wno-unused-parameter -Wno-strict-aliasing -Wno-deprecated -fno-implicit-templates -felide-constructors -MT ha_mroonga_la-ha_mroonga.lo -MD -MP -MF .deps/ha_mroonga_la-ha_mroonga.Tpo -c -o ha_mroonga_la-ha_mroonga.lo `test -f 'ha_mroonga.cpp' || echo './'`ha_mroonga.cpp
/bin/sh ./libtool  --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I.  -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/sql -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/extra/rapidjson/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/regex -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/libbinlogevents/export -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/libbinlogevents/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12 -I/usr/local/Cellar/mysql/5.7.12/include/mysql -I/usr/local/Cellar/groonga/6.0.1/include/groonga -I./lib   -I/usr/local/Cellar/mysql/5.7.12/include/mysql -fno-omit-frame-pointer -DMYSQL_DYNAMIC_PLUGIN -g -O2 -Wall -Wextra -Wno-unused-parameter -Wno-strict-aliasing -Wno-deprecated -fno-implicit-templates -felide-constructors -MT ha_mroonga_la-mrn_table.lo -MD -MP -MF .deps/ha_mroonga_la-mrn_table.Tpo -c -o ha_mroonga_la-mrn_table.lo `test -f 'mrn_table.cpp' || echo './'`mrn_table.cpp
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/sql -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/extra/rapidjson/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/regex -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/libbinlogevents/export -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/libbinlogevents/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12 -I/usr/local/Cellar/mysql/5.7.12/include/mysql -I/usr/local/Cellar/groonga/6.0.1/include/groonga -I./lib -I/usr/local/Cellar/mysql/5.7.12/include/mysql -fno-omit-frame-pointer -DMYSQL_DYNAMIC_PLUGIN -g -O2 -Wall -Wextra -Wno-unused-parameter -Wno-strict-aliasing -Wno-deprecated -fno-implicit-templates -felide-constructors -MT ha_mroonga_la-mrn_table.lo -MD -MP -MF .deps/ha_mroonga_la-mrn_table.Tpo -c mrn_table.cpp  -fno-common -DPIC -o .libs/ha_mroonga_la-mrn_table.o
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/sql -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/extra/rapidjson/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/regex -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/libbinlogevents/export -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/libbinlogevents/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12 -I/usr/local/Cellar/mysql/5.7.12/include/mysql -I/usr/local/Cellar/groonga/6.0.1/include/groonga -I./lib -I/usr/local/Cellar/mysql/5.7.12/include/mysql -fno-omit-frame-pointer -DMYSQL_DYNAMIC_PLUGIN -g -O2 -Wall -Wextra -Wno-unused-parameter -Wno-strict-aliasing -Wno-deprecated -fno-implicit-templates -felide-constructors -MT ha_mroonga_la-ha_mroonga.lo -MD -MP -MF .deps/ha_mroonga_la-ha_mroonga.Tpo -c ha_mroonga.cpp  -fno-common -DPIC -o .libs/ha_mroonga_la-ha_mroonga.o
mrn_table.cpp:582:49: error: no member named 'WARN_LEVEL_WARN' in 'Sql_condition'
        push_warning_printf(thd, Sql_condition::WARN_LEVEL_WARN, 
                                 ~~~~~~~~~~~~~~~^
1 error generated.
make[2]: *** [ha_mroonga_la-mrn_table.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/sql -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/extra/rapidjson/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/regex -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/libbinlogevents/export -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12/libbinlogevents/include -I/private/tmp/mysql20160419-80673-1qijqcp/mysql-5.7.12 -I/usr/local/Cellar/mysql/5.7.12/include/mysql -I/usr/local/Cellar/groonga/6.0.1/include/groonga -I./lib -I/usr/local/Cellar/mysql/5.7.12/include/mysql -fno-omit-frame-pointer -DMYSQL_DYNAMIC_PLUGIN -g -O2 -Wall -Wextra -Wno-unused-parameter -Wno-strict-aliasing -Wno-deprecated -fno-implicit-templates -felide-constructors -MT ha_mroonga_la-ha_mroonga.lo -MD -MP -MF .deps/ha_mroonga_la-ha_mroonga.Tpo -c ha_mroonga.cpp -o ha_mroonga_la-ha_mroonga.o >/dev/null 2>&1
mv -f .deps/ha_mroonga_la-ha_mroonga.Tpo .deps/ha_mroonga_la-ha_mroonga.Plo
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
==> Formula
Tap: mroonga/mroonga
Path: /usr/local/Library/Taps/mroonga/homebrew-mroonga/mroonga.rb
==> Configuration
HOMEBREW_VERSION: 0.9.9
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: f15b631875ef19599bcb4e774d3e67596d1f93ac
Last commit: 27 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 590cc64bd41d628b4cd9afae001432db42b8507b
Core tap last commit: 25 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit haswell
OS X: 10.9.5-x86_64
Xcode: N/A
CLT: 6.2.0.0.1.1424975374
Clang: 6.0 build 600
X11: 2.7.7 => /opt/X11
System Ruby: 2.0.0-p481
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /Users/hhatake/.rbenv/shims/ruby => /Users/hhatake/.rbenv/versions/2.3.0/bin/ruby
Java: 1.8.0_45
==> ENV
HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
MAKEFLAGS: -j4
CMAKE_PREFIX_PATH: /usr/local/opt/openssl:/usr/local
CMAKE_INCLUDE_PATH: /usr/include/libxml2:/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
PKG_CONFIG_PATH: /usr/local/opt/pcre/lib/pkgconfig:/usr/local/opt/msgpack/lib/pkgconfig:/usr/local/opt/openssl/lib/pkgconfig:/usr/local/opt/groonga/lib/pkgconfig:/usr/local/opt/mysql/lib/pkgconfig
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/usr/local/Library/ENV/pkgconfig/10.9
ACLOCAL_PATH: /usr/local/share/aclocal
PATH: /usr/local/Library/ENV/4.3:/usr/local/opt/pkg-config/bin:/usr/local/opt/pcre/bin:/usr/local/opt/openssl/bin:/usr/local/opt/groonga/bin:/usr/local/opt/cmake/bin:/usr/local/opt/mysql/bin:/usr/bin:/bin:/usr/sbin:/sbin

Error: mroonga/mroonga/mroonga 6.01 did not build
Logs:
     /Users/hhatake/Library/Logs/Homebrew/mroonga/01.configure
     /Users/hhatake/Library/Logs/Homebrew/mroonga/01.configure.cc
     /Users/hhatake/Library/Logs/Homebrew/mroonga/02.make
     /Users/hhatake/Library/Logs/Homebrew/mroonga/02.make.cc
     /Users/hhatake/Library/Logs/Homebrew/mroonga/config.log
```

Maybe this error is same as a problem which is encountered in CentOS rpm packaging?
Ful log is here: https://www.dropbox.com/s/1xwj9vv5njidon2/install-mroonga-6.01.log?dl=0

**Comment by kou** [comment]
> Maybe this error is same as a problem which is encountered in CentOS rpm packaging?

Yes.
It'll be fixed in the next release.

**Comment by kenhys** [comment]
This error is fixed in https://github.com/mroonga/mroonga/commit/f79165a29e676400050d6819f84f2498cc1995ff.
When Mroonga 6.01 was released, I 've applied above patch as a workaround.

**Comment by cosmo0920** [comment]
I've confirmed that this issue is fixed in Mroonga 6.02.
